app/scripts/modules/core/src/delivery/delivery.states.ts

```diff
@@ -72,7 +72,7 @@ module(DELIVERY_STATES, [
     url: `?${stateConfigProvider.paramsToQuery(filterModelConfig)}`,
     views: {
       'pipelines': {
-        template: '<executions application="application"></executions>',
+        template: '<executions application="$resolve.app"></executions>',
```

Comment:
Again, this is fine.  There are a few ways to map the resolve to the component input in 1.0 way:

```
pipelines': {
  component: 'executions',
  bindings: { application: 'app' },
}
```
https://ui-router.github.io/ng1/docs/latest/interfaces/ng1.ng1viewdeclaration.html#bindings

Or, change the `executions` input binding to `app` to get the implicit mapping
```
const executionsComponent: IComponentOptions = {
   bindings: {
     application: '<app',
   },
```